app/clients/sms/twilio.py

```diff
@@ -50,17 +54,60 @@ def send_sms(self, to, content, reference, sender=None):
         callback_url = "{}/notifications/sms/twilio/{}".format(
             self._callback_notify_url_host, reference) if self._callback_notify_url_host else ""
         try:
-            message = self._client.messages.create(
-                to=to,
-                from_=from_number,
-                body=content,
-                status_callback=callback_url,
+            # Importing inline to resolve a circular import error when importing at the top of the file
+            from app.dao.service_sms_sender_dao import (
+                dao_get_service_sms_sender_by_service_id_and_number,
+                dao_get_service_sms_sender_by_id
             )
+            messaging_service_sid = None
+            sms_sender_id = kwargs.get("sms_sender_id")
+
+            if sms_sender_id:
```

Comment:
Wouldn't it be less likely to cause an error as-is? That way any 'falsey' value will evaluate as such. Like an empty string, for example.

Reply:
It's more likely to hide a problem rather than reveal the problem with an error.  None is not a boolean and should not be treated as such.  Elsewhere in my feedback, I included a link to Pep8, which discusses this issue.

What if the sender ID is 0?
```
$ python3
Python 3.8.10 (default, Jun 22 2022, 20:18:18) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> bool(0)
False
```

Reply:
Good point. I'll change it to `is not None`